Cartfile

```diff
@@ -1,7 +1,3 @@
-github "Hearst-DD/ObjectMapper" ~> 1.0
 github "Alamofire/Alamofire" ~> 3.0
 github "bignerdranch/Freddy" ~> 2.0
-github "daltoniam/Starscream" ~> 1.0
-
-# ObjectMapper is temporarily required for AlchemyLanguage.
-# It can be removed after AlchemyLanguage is ported to use the RestKit architecture.
\ No newline at end of file
+github "daltoniam/Starscream" ~> 1.0
```

Comment:
Yay! I forgot that we still have ObjectMapper in the Cartfile. Looking forward to this change.